xcode/Subconscious/Shared/Services/Noosphere/Sphere.swift

```diff
@@ -64,8 +63,66 @@ public protocol SphereProtocol {
     func traverse(petname: Petname) throws -> Sphere
 }
 
-protocol SphereIdentityProtocol {
```

Comment:
No longer used anywhere, so I removed it. Would prefer to make this part of SphereProtocol anyway.